### PR TITLE
9 Norwegian Moneropedia entries (C–D)

### PR DESCRIPTION
**Pull request by MindWallet**

### CODE REVIEW

File: _i18n/nb-no/resources/moneropedia/canonically-unique-host.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+tags: ["kovri"]
+terms: ["Kanonisk-unik-vert"]
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/canonically-unique-host.md:3]: Add English term

File: _i18n/nb-no/resources/moneropedia/canonically-unique-host.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+### Inngående informasjon
+
+En kanonisk-unik-vert er definert av eksterne, autoritative kilder; vanligvis gjennom en [DNS](https://en.wikipedia.org/wiki/DNS). Når vertsnavnet til en motpart avklares, må man mest sannsynlig bruke en ekstern kilde for oppklaringen, med mindre man har følgende implementert:
+
+- en databasefil tilsvarende en [vertsfil](https://en.wikipedia.org/wiki/Hosts_(file))
+- et løsningsprogram for interne nettverk (som etter hvert foretar trekkforespørsler fra eksterne kilder)
+
+### Merknader
+
+- Monero bruker primært en kanonisk-unik-vert-resolusjon, mens I2P bare bruker lokal-unik-vert-resolusjon.
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/canonically-unique-host.md:21]: Perhaps add "en" after "bruker" in the last part for consistency

File: _i18n/nb-no/resources/moneropedia/change.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+---
+
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Det grunnleggende
+
+Monero som sendes som del av en transaksjon som tilbakeføres til kontoen din i stedet for å gå til en annen mottaker.
+
+### Mer informasjon
+
+Lommeboken i Monero-programvaren lager veksel automatisk, men når du sender en transaksjon, tar du en inndata som du kontrollerer og forteller Monero-nettverket hva som skal gjøre med det. Inndataen er et «innskudd» til kontoen din som du kan forbruke. Utdataene er delen av transaksjonen som forteller Monero-nettverket hvor midlene skal sendes til.
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/change.md:13]: typo: gjøres*

File: _i18n/nb-no/resources/moneropedia/clearnet.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- din [ISP](https://en.wikipedia.org/wiki/ISP)
+- nettsiden/tjenesten/personen du kommuniserer med
+- muligens en [Five Eyes](https://en.wikipedia.org/wiki/5_Eyes)-kapabel enhet
+
+Og selv om du bruker [HTTPS](https://en.wikipedia.org/wiki/HTTPS) eller liknende (som *krypterer* overføringen din), er ruten din verken skjult eller anonym. Den er altså «in the *clear*» (skjult og utenfor fare).
+
+### Inngående informasjon
+
+Fordi en tradisjonell [VPN](https://en.wikipedia.org/wiki/VPN) ikke kan redde deg fra clearnet (i og med at du fremdeles bruker *clearnet* (selvom du er mer sikret enn uten en VPN)), bør du bruke et *anonymt overleggsnettverk* for å unngå å bruke clearnet direkte:
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/clearnet.md:20]: Typo: selv om*

File: _i18n/nb-no/resources/moneropedia/clearnet.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- Kovri
+- Java-I2P
+- [Tor](https://torproject.org/)
+
+Disse teknologiene beskytter deg fra clearnet ved å bygge et anonymt nettverk **over** clearnet med hensikt å holde overføringene dine både krypterte **og** anonyme.
+
+Her er et nøyaktig, [interaktivt diagram](https://www.eff.org/pages/tor-and-https) gitt av [EFF](https://www.eff.org/) som beskriver *clearnet* i sammenheng med **Tor**. Konseptet gjelder også (i en viss grad) Kovri og I2P hva gjelder anonymitet, med unntaket av at:
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/clearnet.md:28]: Konseptet gjelder også (i en viss grad) for*

File: _i18n/nb-no/resources/moneropedia/cryptocurrency.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+---
+terms: ["cryptocurrency", "cryptocurrencies", "altcoin", "altcoins","kryptovaluta", "kryptovalutaer", "altcoin", "altcoins"]
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/cryptocurrency.md:2]: spacing error

File: _i18n/nb-no/resources/moneropedia/cryptocurrency.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+### Det grunnleggende
+
+En digital valuta som bruker krypteringsteknikker for å regulere genereringen av valutaenheter og verifisere overføringen av midler, som vanligvis opererer uavhengig av en sentralbank.
+
+### Mer informasjon
+
+Kryptovaluta er et generisk begrep for et stort sett med digitale eiendeler som bruker krypteringsteknikker for å generere valutaenheter, verifisere transaksjoner og overføre verdi. Kryptovalutaer betraktes generelt som å være desentraliserte og bør ikke forveksles med virtuell valuta som er en form for digitale penger som vanligvis kontrolleres av sine skapere eller utviklere. Noen eksempler på virtuell valuta er gametime i World of Warcraft, ROBUX i Roblox, bonuspoengprogram eller Ripple, der alle kan veksles inn mot valuta eller kontanter, men ikke betraktes som kryptovaluta fordi de er sentraliserte og kontrollert/utstedet av en enkel entitet.
+
+Monero er én av mange kryptovalutaer som er tilgjengelig. Andre eksempler er Bitcoin, Litecoin, Dogecoin, Dash, Zcash, osv, men nesten alle andre kryptovaluta mangler funksjoner som gjør dem til ekte penger (og viktigst av alt fungibilitet, som er et krav for at det kan kalles verdioppbevaringsmiddel).
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/cryptocurrency.md:15]: .. kan kalles for et*

File: _i18n/nb-no/resources/moneropedia/denominations.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
+---
+terms: ["denominations", "subunits", "valører", "underenheter", "tacoshi", "piconero", "nanonero", "micronero", "millinero", "centinero", "decinero","decanero","hectonero","kilonero","meganero","giganero"]
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/denominations.md:2]: fix spacing

File: _i18n/nb-no/resources/moneropedia/denominations.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+| micronero  | 10^-6    | 0,000001          |
+| millinero  | 10^-3    | 0,001             |
+| centinero  | 10^-2    | 0,01              |
+| decinero   | 10^-1    | 0,1               |
+|============+==========+===================|
+| **monero** | **10^0** | **1**             |
+|============+==========+===================|
+| decanero   | 10^1     | 10                |
+| hectonero  | 10^2     | 100               |
+| kilonero   | 10^3     | 1 000             |
```
Comment (StineeH) [inline _i18n/nb-no/resources/moneropedia/denominations.md:29]: 1 000 -> 1000*

**Review by erciccione — APPROVED** [review]
Needs to be squashed.

**Review by erciccione — CHANGES_REQUESTED** [review]
'@' tags need to be reintroduced.

**Review by erciccione — APPROVED** [review]
There are some tags that point to entries that got removed, but doesn't matter, i will fix them in bulk later on.

**Pull request merged**